doc/content/microreactors/airjacket/index.md

```diff
@@ -0,0 +1,502 @@
+output obtained.
+
+The Boussinesq approximation is applied as a forcing term in the `userf` routine. Note that the Richardson number is calculated in the
+`userchk` subroutine, and transferred to `userf` using a `common` block, as shown below. Complicated calculations that require global
+operations across all MPI ranks are best performed in `userchk`, as subroutines like `userbc`, `useric`, and `userf` are not called
+by every MPI process by design, as mentioned in the Nek5000 documentation and template `.usr` file (`Nek5000/core/zero.usr`). The Boussinesq term is gradually
+ramped down in the upper plenum, as we want the highly biased flow to redistribute and exit the domain cleanly, without causing
+backflow at the outlet, which can cause the simulation to diverge.
+
+!listing microreactors/gcmr/airjacket/airjacket.usr start=subroutine userf end=c---- include-end=False
+
+### Meshing
+
+When meshing, we performed h-refinement in our meshing software (Cubit) by adding elements to areas of transition between the channel and the plena, and other
+regions where under-resolution was observed. Then the final mesh refinement was performed using p-refinement in Nek (using the `lx1` parameter in the `SIZE` file), as is typical
+in spectral elements. Also of note are the boundary layers, which can have a relatively aggressive growth rate in spectral elements([boundary_layer]).
+
+!media airjacket/boundary_layer.png
+       style=width:100%
+       id=boundary_layer
+       caption=The boundary layers in the channel.
+
+The mesh was created using Cubit and a `stp` file provided by the reactor designer. The file was exported to the Exodus format and
+converted to the Nek-supported `.re2` using the utility `exo2nek`. If `Nek5000/bin` is added to the `$PATH`, `exo2nek` can be executed
+directly from the command line. The tool guides the user through the mesh conversion process. In this case, we recommend skipping the element
+right-handedness check for the mesh and proceeding with the mesh conversion process as suggested by the command line prompts.
+
+Wall resolution can be verified by checking $y+$ values call the `print_yplimits` subroutine (provided in the `extract_yp.f` file)
+in `userchk` at whatever frequency is desired.
+The subroutine calculates $y+$ values and outputs a variety of relevant metrics, the most important being the ratio of wall points
+with $y+ \leq 1$ to the total number of wall points, outputted as `ratio less than 1.0 is :` in the `logfile`. Ideally, this ratio
+should be nearly 1.0. The $y+$ values can also be visualized using the output array from `print_yplimits` dumped into a Nek5000
+output file using the `outpost` function.
+
+### Statistics at runtime and postprocessing
+
+To iterate on the Reynolds number until the correct pressure drop of nearly zero (less than the pressure solver tolerance) is obtained,
```

Comment:
pressure drop is not zero?
do you mean the error to the target P drop?

Reply:
yes, thanks. edited to clarify